test/unittest/logging/CMakeLists.txt

```diff
@@ -39,5 +49,76 @@ if(NOT ((MSVC OR MSVC_IDE) AND EPROSIMA_INSTALLER))
                 )
         endif()
         add_gtest(LogTests SOURCES ${LOGTESTS_SOURCE})
+
+        set(LOGFILETESTS_SOURCE
+            ${PROJECT_SOURCE_DIR}/src/cpp/xmlparser/XMLProfileManager.cpp
```

Comment:
Too much code files to an unit test. Maybe if you separate th logging XML part to a separate CPP, we don't have to depend one dynamic type and so on..

Reply:
Another alternative is the xml testing part go to the unit testing of xmlparser.

Reply:
I have split the tests between the log management and log parsing, moving the second group to XMLProfileTests